includes/compat/mailchimp-for-woocommerce.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Newspack setup
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+function newspack_mc4c_remove_cookie() {
```

Comment:
Linter has [two issues](https://app.circleci.com/pipelines/github/Automattic/newspack-plugin/575/workflows/1376b9e3-6ca7-43c5-89c6-72aca978a5ed/jobs/2449/parallel-runs/0/steps/0-102) with this file.